Doc/library/uuid.rst

```diff
@@ -434,6 +436,14 @@ Here are some examples of typical usage of the :mod:`uuid` module::
    >>> uuid.MAX
    UUID('ffffffff-ffff-ffff-ffff-ffffffffffff')
 
+   >>> # get UUIDv7 creation (local) time as a timestamp in millisecond
+   >>> u = uuid.uuid7()
+   >>> u.time  # doctest: +SKIP
+   1743936859822
+   >>> # get UUIDv7 creation (local) time as a datetime object
+   >>> datetime.fromtimestamp(u.time / 1000)  # doctest: +SKIP
+   datetime.datetime(...)
```

Comment:
Let's include the import to make clear [_which_ `datetime`](https://adamj.eu/tech/2019/09/12/how-i-import-pythons-datetime-module/):

```suggestion
   >>> import datetime as dt
   >>> dt.datetime.fromtimestamp(u.time / 1000)  # doctest: +SKIP
   datetime.datetime(...)
```

But this doesn't work, do you mean something else?

```pytb
>>> dt.datetime.fromtimestamp(u.time / 1000)
Traceback (most recent call last):
  File "<python-input-11>", line 1, in <module>
    dt.datetime.fromtimestamp(u.time / 1000)
    ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
ValueError: year must be in 1..9999, not 12525756
```

Reply:
Wait. How come it fails?

Reply:
On Windows (and Python 3.11, agreed it could be different):

```py
>>> import datetime as dt
>>> dt.datetime.fromtimestamp(1743936859822 / 1000)
datetime.datetime(2025, 4, 6, 12, 54, 19, 822000)
```

And `1743936859822` should be the correct milliseconds timestamp.

Reply:
Aha, that failure was with the last alpha 6:

```pycon
❯ python3.14
Python 3.14.0a6 (v3.14.0a6:77b2c933cab, Mar 14 2025, 11:45:56) [Clang 16.0.0 (clang-1600.0.26.6)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> u = uuid.uuid7()
>>> u.time
395212535438381227
```

Works fine with latest `main` :)

```pycon
❯ ./python.exe
Python 3.14.0a6+ (heads/main:316974382af, Apr  7 2025, 15:04:45) [Clang 16.0.0 (clang-1600.0.26.6)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> u = uuid.uuid7()
>>> u.time
1744037912688
>>> import datetime as dt
>>> dt.datetime.fromtimestamp(u.time / 1000)
datetime.datetime(2025, 4, 7, 17, 58, 32, 688000)
```